.eslintrc

```diff
@@ -8,6 +8,10 @@
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
+  "rules": {
+    "no-use-before-define": "off",
+    "no-unused-vars": "off"
```

Comment:
I'm not sure I follow which things in your test needed this.  The `declare namespace Cypress` thing perhaps?

Reply:
With these turn off, the eslint errors I get are:

```
    5:14  error  'Cypress' was used before it was defined  no-use-before-define
    5:44  error  'Cypress' was used before it was defined  no-use-before-define
    6:5   error  'Cypress' was used before it was defined  no-use-before-define
    6:42  error  'Cypress' was used before it was defined  no-use-before-define
  124:23  error  'Subject' is defined but never used       no-unused-vars
  129:22  error  'selector' is defined but never used      no-unused-vars
  129:40  error  'value' is defined but never used         no-unused-vars
```

I believe this is related to the mix of tsc + commonjs + cypress that we're using.  I don't think these linting errors are correct.

Thoughts?

Reply:
Adding `/* global Cypress */` to the top results in:

`error  'Cypress' is already defined as a built-in global variable`

So somehow it's already defined as a global and is used before it's defined.

Reply:
Weird.  Yeah, I'm fine with the change.